packages/@cdk-cosmos/core/src/pattern.ts

```diff
@@ -0,0 +1,56 @@
+import { Construct } from '@aws-cdk/core';
+
+type Dictionary = { [key: string]: string };
+type Scope = {
+  Name: string;
+  Partition?: string;
+  Cosmos?: Scope;
+  Galaxy?: Scope;
+};
+
+export const PATTERN = {
```

Comment:
List of all the pattens we support

Reply:
In the Future these might come from envs, for Dev mode to allow multiple cosmos' in once account for testing...